bazel/brotli.patch

```diff
@@ -0,0 +1,31 @@
+# https://github.com/google/brotli/pull/839
+diff --git a/c/enc/bit_cost.h b/c/enc/bit_cost.h
+index 6586469..31838f1 100644
+--- a/c/enc/bit_cost.h
++++ b/c/enc/bit_cost.h
+@@ -45,7 +45,7 @@ static BROTLI_INLINE double BitsEntropy(
+     const uint32_t* population, size_t size) {
+   size_t sum;
+   double retval = ShannonEntropy(population, size, &sum);
+-  if (retval < sum) {
```

Comment:
Can you please upstream this patch? We _heavily_ discourage custom patching during external dep fetch.

Reply:
This patch has been already merged upstream: https://github.com/google/brotli/pull/839. Also https://github.com/google/brotli/pull/840. I put the references to the patch file as comments.

Reply:
I've dropped the patch. Now the dep fetches from the main dev branch.

Reply:
awesome